.gitignore

```diff
@@ -1,6 +1,6 @@
 # Ignore everything on the config folder, except sample and default files
 config/*
-!config/defaults.yml
+!config/*-default*.yml
```

Comment:
The `!config/defaults.yml' could stay.
```
!config/defaults.yml
!config/*-default.yml
```
?

Reply:
Ah, yes! Well spotted. Thanks.